draft-ietf-quic-http.md

```diff
@@ -95,40 +95,84 @@ It's not shouting; when they are capitalized, they have the special meaning
 defined in {{!RFC2119}}.
 
 
-# QUIC advertisement
+# QUIC Advertisement
 
-A server advertises that it can speak HTTP/QUIC via the Alt-Svc HTTP response 
-header. It does so by including the header in any response sent over a non-QUIC 
-(e.g. HTTP/2) connection: 
+A server advertises that it can speak HTTP/QUIC via the Alt-Svc ({{!RFC7838}}) 
+HTTP response header, using the ALPN token defined in 
+{{connection-establishment}}. 
 
-   Alt-Svc: quic=":443"
+Thus, a server could indicate in an HTTP/1.1 or HTTP/2 response that HTTP/QUIC
+was available on UDP port 443 by including the following header in any
+response:
 
-In addition, the list of QUIC versions supported by the server can be specified
-by the v= parameter.  For example, if a server supported both version 33 and 34
-it would specify the following header:
+    Alt-Svc: hq=":443"
 
-   Alt-Svc: quic=":443"; v="34,33"
+## QUIC Version Hints {#alt-svc-version-hint}
 
-On receipt of this header, a client may attempt to establish a QUIC connection
-on port 443 and, if successful, send HTTP requests using the mapping described
-in this document.
+This document defines the "v" parameter for Alt-Svc, which is used to provide
+version-negotiation hints to HTTP/QUIC clients. Syntax:
+
+    v = quoted-versions
+    quoted-versions = DQUOTE *( version-string "," ) version-string DQUOTE
+    version-string = token; percent-encoded QUIC version
+
+For example, if a server supported both version Q033 and Q034 it would specify
+the following header:
+
+    Alt-Svc: hq=":443"; v="Q034,Q033"
```

Comment:
A note to discuss version representation:

You have a character string, I chose to go with just numbers.  Both have merits.  Numbers are simpler, but potentially longer (hexadecimal saves a few octets for larger values if you want to go there), but a four character string can be quite friendly.  You have to pick the right values though or you get a mash of %xx strings.

`Alt-Svc: hq=":443;v=<version>;v=<version>` might be better than creating a micro-syntax for the list.  See also httpwg/http-extensions#281